mesh_segmenter/test/utest.cpp

```diff
@@ -42,30 +39,32 @@ TEST(SegmentationTest, TestCase1)
   seg.segmentMesh();
   std::vector<vtkSmartPointer<vtkPolyData> > meshes = seg.getMeshSegments();
 
+  #ifdef NDEBUG
+  // release build stuff goes here
+  #else
   // Display meshes for debugging
-  if (g_display_meshes)
-  {
-    vtk_viewer::VTKViewer viz;
+  vtk_viewer::VTKViewer viz;
```

Comment:
since there isn't a use for the `#ifdef NDEBUG` case could we use `#ifdef DEBUG` instead?

Reply:
NDEBUG is defined by the C/C++ standard, while DEBUG or _DEBUG are left to compiler or user implementation.  We would have to define DEBUG ourselves, which may conflict with others' definitions, while _DEBUG varies in what it changes.

But I don't have strong feelings.  Your call.

Reply:
I think I may not be interpreting this right, when one builds in debug node `-DCMAKE_BUILD_TYPE=Debug` then the compiler defines the `NDEBUG` flag which is accessible in code right?

Reply:
@jrgnicho 
I have simplified the #ifdef ... #else to #ifndef.